server/routerlicious/packages/lambdas-driver/src/kafka-service/runner.ts

```diff
@@ -113,6 +113,10 @@ export class KafkaRunner implements IRunner {
 		this.stopped = true;
 		Lumberjack.info("KafkaRunner.stop starting.");
 		try {
+			this.runnerMetric.setProperties({
+				caller,
```

Comment:
If I remember correctly, `setProperties` is quite expensive, it will:
1. create an object in caller
2. deconstruct object
3. check key existence
4. add key if not exist.

Any reasons why not juse use `setProperty` instead? It will make caller side cleaner too

Reply:
I wasnt aware that setProperties is expensive, I can certainly use setProperty instead.